e2e/package.json

```diff
@@ -21,6 +21,10 @@
     "@axe-core/playwright": "4.8.5",
     "sinon": "17.0.0"
   },
-  "author": "NASA Ames Research Center",
+  "homepage": "https://github.com/nasa/openmct/tree/master/e2e",
```

Comment:
let's remove the `homepage` prop since this is our e2e testing framework and it will not be served